fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/exceptions/ErrorCode.java

```diff
@@ -122,6 +122,7 @@ public enum ErrorCode {
     INVALID_TABLE_DEFINITION("42F16"),
     UNKNOWN_TYPE("42F18"),
     INVALID_RECURSION("42F19"),
+    INCOMPATIBLE_TABLE_ALIAS("42F20"),
```

Comment:
Hard to search the PR for usages of this new code. But where is it used?

Reply:
`TypeUtils.java`.